<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RbkitMainWindow</class>
 <widget class="QMainWindow" name="RbkitMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>840</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rbkit - A Ruby Profiler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="chartingTab">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_Rbkit"/>
   </widget>
   <widget class="QMenu" name="menu_Memory_Profiling">
    <property name="title">
     <string>&amp;Memory Profiling</string>
    </property>
    <addaction name="action_Trigger_GC"/>
    <addaction name="actionHeap_Snapshot"/>
    <addaction name="actionCompare_Heapsnapshots"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Memory_Profiling"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Connect"/>
   <addaction name="action_Trigger_GC"/>
   <addaction name="actionHeap_Snapshot"/>
   <addaction name="actionCompare_Heapsnapshots"/>
  </widget>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="../tool_icons.qrc">
     <normaloff>:/icons/connect-48.png</normaloff>:/icons/connect-48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Rbkit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About_Rbkit">
   <property name="text">
    <string>&amp;About Rbkit</string>
   </property>
  </action>
  <action name="action_Trigger_GC">
   <property name="icon">
    <iconset resource="../tool_icons.qrc">
     <normaloff>:/icons/startgc-48.png</normaloff>:/icons/startgc-48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start GC</string>
   </property>
  </action>
  <action name="actionHeap_Snapshot">
   <property name="icon">
    <iconset resource="../tool_icons.qrc">
     <normaloff>:/icons/snapshot-48.png</normaloff>:/icons/snapshot-48.png</iconset>
   </property>
   <property name="text">
    <string>Heap Snapshot</string>
   </property>
   <property name="toolTip">
    <string>Record heapsnapshot</string>
   </property>
  </action>
  <action name="actionCompare_Heapsnapshots">
   <property name="icon">
    <iconset resource="../tool_icons.qrc">
     <normaloff>:/icons/Compare-48.png</normaloff>:/icons/Compare-48.png</iconset>
   </property>
   <property name="text">
    <string>Compare  Snapshots</string>
   </property>
   <property name="toolTip">
    <string>Compare Heapsnpshot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../tool_icons.qrc"/>
 </resources>
 <connections/>
</ui>
